docs/sources/k6-studio/record-your-first-script.md

```diff
@@ -37,44 +37,49 @@ To create a test recording:
 
 1. Open the k6 Studio desktop application.
 1. Click **Record Flow**.
-1. Type "test-api.k6.io" under **Starting URL**.
+1. Type "quickpizza.grafana.com" under **Starting URL**.
```

Comment:
Side-note: I find the URL incredibly long, wish it was just `pizza.grafana.com` or something 🫤